README.md

```diff
@@ -1 +1,47 @@
-# Riyaz-Portfolio
\ No newline at end of file
+# Riyaz-Portfolio
+
+## Built With
+
+HTML
+CSS
+
+#### Getting Started
+
+https://r-ahmed2022.github.io/Riyaz-Portfolio/
+
+#### Getting Started
+
+Instructions on how to access my project locally. To Access my project locally please follow the following instructions;
+
+Enter this url: https://github.com/r-ahmed2022/Riyaz-Portfolio in your web browser.
+Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side.
+Select download Zip option from drop down menu.
+
+## Prerequisites
+
+-> Setup GitHub repository
+-> Install git
+-> Setup Linters
+. Lighthouse
+. Weblint
+. Stylelint
+
+Deployment
+Authors
+👤 Riyaz
+
+GitHub: @https://github.com/r-ahmed2022/
+LinkedIn: https://www.linkedin.com/in/riyaz-ahmed
+
+### Contributing
+
+#### Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+Give a ⭐️ if you like this project!
+
+# Acknowledgments
+
+Hat tip to anyone whose code was used
+Inspiration
+etc
```

Comment:
[OPTIONAL] Please add some actual acknowledgments, you could mention Microverse and the Figma template design here. Having this section with generic messages like these makes your project look unprofessional.

Reply:
done as advised